### PR TITLE
[ADD] repairs: add repair orders doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [ADD] adding a `Repairs` section to Odoo's documentation, plus a `repair_orders` doc, specific to 17.0.

Here's what I did:
- Added `Repairs` to file directory (nested under Inventory & MRP section)
- Wrote new repair orders doc for 17.0
- Added in 17.0 UI-specific screenshots
- Added new doc to appropriate area of toc tree

[Project Task Link](https://www.odoo.com/web#id=3862548&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hey @AntoineVDV I'm adding a new Repairs section to the inventory_and_mrp directory and the codeowners need to be updated. I am part of the Materials scope documentation team. Thank you!

**Comment by brse-odoo** [comment]
Hi @Felicious, this PR is ready for your review when you have a chance! Thank you

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:14]: Since this is a complicated workflow with many steps, how about outlining the steps, with links to relevant sections at the top here before we jump in? It could give the reader a big picture summary so they can keep track of all the steps. For example,

1. Sell and deliver product
2. Customer returns damaged product (ref link to "Return order" section of your doc)
3. Create and process RO (link to "Create Repair Order" section of your doc)
4. Re-deliver product to customer (link to "Return product to customer")

**Review by Felicious — CHANGES_REQUESTED** [review]
Hi @brse-odoo,

AMAZING job with this doc, Brandon. With *two* returns in the workflow, this doc could easily be confusing, but the steps were easy to follow. I can tell you did significant work on this. :blush: 

Most of my suggestions were really minor wording suggestions, but I ran into a minor issue finding the return document in the repair order. If my comment doesn't quite make sense, we can work offline to determine what causes the return order to not appear in the search. That is the only change we must address before moving this doc onto the next steps.

After we've figured it out, re-tag me for a second review! It'll be a swift review, I promise :smile:

**Comment by AntoineVDV** [comment]
> Hey @AntoineVDV I'm adding a new Repairs section to the inventory_and_mrp directory and the codeowners need to be updated. I am part of the Materials scope documentation team. Thank you!

Hello, I created the https://github.com/orgs/odoo/teams/repairs-doc-review team and set you maintainer so you can invite more people if needed. The codeowners are adapted to request reviews from this team when a change is made in the new Repairs doc ;)

(I unsubscribed from this thread so tag me if there is a problem)

File: content/applications/inventory_and_mrp/repairs.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs.rst:10]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:19]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:79]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:83]: It looks like the product you used for the return was [FURN_8855] Drawer, and on the repair you listed a different product [FURN_2100] Drawer Black. That's why it didn't show up

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:109]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:138]: I chose the term I did because it's what it read when I inspected the code, but now that I know it should be slider icon, I'll use that from now on. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:150]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:165]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:183]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:189]: Nice catch, small detail I overlooked with the wording I chose. Changed it accordingly. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:200]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:14]: Added ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:93]: oh my goodness *face palm*. Thank you for clarifying. I'll try it again!

Comment (Felicious) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:93]: Tried again with the correct product selected, and it works as expected! Thank you for clarifying for me!

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:97]: Totally up to you how you'd like to describe this, but here's a suggestion!
```suggestion
Tick the :guilabel:`Under Warranty` checkbox, if the product being repaired is covered
by a warranty. If ticked, the :guilabel:`Customer` is not charged for all the parts used in the repair
order.
```

Is this still correct? I wanted to clarify your original sentence:
> If ticked, the sales price is set to `0` for all products transferred from the repair
order.

**Review by Felicious — APPROVED** [review]
Hi @brse-odoo! I confirmed that the part I couldn't test during my last review works as expected. Great job with this doc! (:

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:97]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
Alright @odoo/us-doc-review, this PR is ready for the final round of content review. Thank you in advance!

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:102]: Optional:
I don't think this screenshot  is as informative as a fully completed RO form, to visually demonstrate for the reader what the written context is describing. Would consider switching 😉

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:133]: It's clear to me that this is what these options are for— the bigger question I have is what exactly are we doing on the _Parts_ tab? What's it for? For what purpose would we Add, Remove, or Recycle parts here?

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:199]: Optional: 
I like this shot showing the difference between the locations— consider zooming out to show more context around this section. By itself, it's isolated and isn't clear where exactly we're looking at this in Odoo.

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:210]: Since the instructions for products not under warranty is completely different then the steps that proceed after this section, consider putting an h4 heading here as well as line 218 to differentiate the two instructions as conditional flows.

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Look like we have our first repairs doc 🙌 
Approving with comments. Please see below, and tag SALI when you're ready for merge, thanks!

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:102]: Replaced screenshot with zoomed-out RO image, showing everything filled out. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:199]: Agreed - replaced screenshot with zoomed-out one showing all relevant areas of the page. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:210]: Nice catch - added H4 headings here to separate the instances of customer returns. Suggestion implemented ✅

**Comment by brse-odoo** [comment]
> Hi @brse-odoo Look like we have our first repairs doc 🙌 Approving with comments. Please see below, and tag SALI when you're ready for merge, thanks!

Hi @samueljlieber, this PR has passed us-doc and is ready for merge, thank you!

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:153]: Small update to the icon name :)
image
```suggestion
To add additional columns to the line, click the :guilabel:`(optional columns drop-down)` icon, at
the far-right of the header row. Select the desired options to add to the line.
```

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/repairs/repair_orders.rst:168]: Just double checking- should there be a space here?
```suggestion
default, this is set to :guilabel:`Your Company: Repairs`, indicating this is a repair type
```

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo! Awesome work with this brand new Repairs doc and section! Your changes in this PR look good to me, I have just two small things to point out– please see below!

Approving and delegating to you to merge at your discretion! Thank you :)
.....
@robodoo delegate=brse-odoo

File: content/applications/inventory_and_mrp/repairs/repair_orders.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/repairs/repair_orders.rst:168]: There's not a space, although I know it looks weird :-/ But that's how it appears in the UI

**Pull request closed without merging**
